src/components/Loading/Loading.scss

```diff
@@ -12,13 +18,11 @@
   // gray-dark #43484e in hex.
   --faded-gray-dark: 67, 72, 78;
   background-color: rgba(var(--faded-gray-dark), 0.75);
-  // TODO: The above can be replaced with the below when Firefox supports "Relative RGB colors".
-  //background-color: rgb(from var(--gray-dark) / 0.75);
```

Comment:
I understood this comment to mean that we should replace Line14 
`background-color: rgba(var(--faded-gray-dark), 0.75);`

 with the commented out line
`background-color: rgb(from var(--gray-dark) / 0.75);`

Reply:
Yes this is updated after I changed the color-notation stylelint  rule like we discussed